### DISCUSSION TITLE
I published a Quarto extension on Github. Should I reference it somewhere?

### DISCUSSION THREAD

**Discussion by pagiraud** [quarto-dev/quarto-cli#9962] [Q&A] [answered]
### Description

I just published, partly thanks to the help of some of you, a little Quarto extension and Pandoc filter. My purpose is not to "advertise" it, but should I reference it somewhere so that people may find it if they look for such an extension?

**Comment by mcanouil** [comment, marked as answer]
See:
- https://quarto.org/docs/extensions/
- https://github.com/quarto-dev/quarto-cli/discussions/categories/show-and-tell
- https://m.canouil.dev/quarto-extensions/

**Reply by pagiraud** [reply]
Thanks for this reply! What thing are the “GitHub topics” you refer to in the third link?

**Reply by mcanouil** [reply]
See https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/classifying-your-repository-with-topics

**Reply by pagiraud** [reply]
Thanks! I just followed your advices.
